FluentUI.Demo/src/androidTest/java/com/microsoft/fluentuidemo/demos/V2ProgressIndicatorUITest.kt

```diff
@@ -0,0 +1,89 @@
+package com.microsoft.fluentuidemo.demos
+
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.semantics.ProgressBarRangeInfo
+import androidx.compose.ui.test.assertHeightIsEqualTo
+import androidx.compose.ui.test.assertRangeInfoEquals
+import androidx.compose.ui.test.assertWidthIsEqualTo
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.controlTokens.CircularProgressIndicatorSize
+import com.microsoft.fluentui.theme.token.controlTokens.ShimmerShape
+import com.microsoft.fluentui.tokenized.progress.CircularProgressIndicator
+import com.microsoft.fluentui.tokenized.progress.LinearProgressIndicator
+import com.microsoft.fluentui.tokenized.progress.Shimmer
+import org.junit.Rule
+import org.junit.Test
+
+class V2ProgressIndicatorUITest {
```

Comment:
Any accessibility test can be added?

Reply:
unfortunately for compose views, there is no accessibility framework. It should be manually done.